### DISCUSSION TITLE
Differences in `*.dist-info/METADATA` files

### DISCUSSION THREAD

**Discussion by UebelAndre** [pypa/setuptools#3181] [Q&A]
Hi, when I run the following on two different machines, I get two different `METADATA` files and I don't understand why or how to get them to match.

`requirements.txt`
```text
future==0.18.2
```
```shell
python3 -m pip wheel --isolated -r requirements.txt
```

The content of `future-0.18.2.dist-info/METADATA` differ where the file on one host has `License-File: LICENSE.txt` and the other one doesn't. As far as I can tell, both hosts have the versions of what I think are the only packages being used here:

```shell
python3 -m pip --isolated list
```
```text
Package     Version
----------- -------
pip         22.0.4
setuptools  60.10.0
wheel       0.37.1
```

What is actually in charge of the layout of `*.dist-info/METADATA` files? And why would these files be different on these different hosts? Any help would be greatly appreciated.

**Comment by abravalheri** [comment]
Hi @UebelAndre, I suspect that running `pip wheel --isolated` ignores the version of setuptools that you have installed in your machine and then try to install it again inside an isolated virtual environment to follow PEP 517...

Any chance that would be the case?
